### ISSUE TITLE
Evaluating on the train sets

### ISSUE THREAD

**Issue by isaaccorley** [wangzhecheng/SkyScript#4]
First I want to thank the authors for the hard work on generating the dataset! 

I noticed in the `benchmark_dataset_info.py` script that it appears the evaluation for 4/8 datasets is being done on the train sets and not the test sets. Any particular reason why?

For the datasets with a test set like AID and EuroSAT could you also explain the splitting process used?

**Comment by vietvo89** [comment]
Hey @isaaccorley, have you successfully run the zero-shot evaluation script? Or did you load the pre-trained model and evaluate it successfully? I tried to run the notebooks provided using Colab. But I couldn't load the model. The error I got is described on https://github.com/wangzhecheng/SkyScript/issues/2#issuecomment-2856999267

Very appreciate if you could help. Thanks
